modules/zenith-public/sql/TraitsAndJAs.sql

```diff
@@ -0,0 +1,362 @@
+UPDATE `abilities` SET `level` = 71, `recastTime`= 90, `content_tag`= NULL WHERE `name` = 'restoring_breath'; -- Recast changed from 60 to 90
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'dragon_breaker'; -- TODO: Duration changed from 180 to 120
+
+-- SMN
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'mana_cede';
+
+-- BLU
+UPDATE `abilities` SET `level` = 65, `content_tag`= NULL WHERE `name` = 'efflux';
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'unbridled_learning';
+
+-- COR
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'triple_shot'; -- TODO: Potency changed from 40% to 10%
+
+-- PUP
+UPDATE `abilities` SET `level` = 60, `content_tag`= NULL WHERE `name` = 'cooldown';
+-- UPDATE `abilities` SET `level` = 71, `content_tag`= NULL WHERE `name` = 'tactical_switch'; -- TODO: Not yet implemented
+
+-- DNC
+UPDATE `abilities` SET `level` = 71, `recastTime`= 30, `content_tag`= NULL WHERE `name` = 'presto'; -- TODO: Potency changed from 5 steps to original 3. Daze increase from 5 to original 2. Step Accuracy from +50 to +20.
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'feather_step';
+-- UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'climactic_flourish'; -- TODO: Not yet implemented. TODO: Flourish III Category does not display in the menu when making them available at an earlier level.
+-- UPDATE `abilities` SET `level` = 71, `recastTime`= 45, `content_tag`= NULL WHERE `name` = 'striking_flourish'; -- TODO: Not yet implemented
+-- UPDATE `abilities` SET `level` = 75, `recastTime`= 60, `content_tag`= NULL WHERE `name` = 'ternary_flourish'; -- TODO: Not yet implemented
+
+-- SCH
+-- UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'libra'; -- TODO: Not yet implemented.
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'perpetuance';
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'immanence';
+
+-- Strategems
+UPDATE `abilities_charges` SET `level` = 65 WHERE `recastId` = 231 AND `maxCharges` = 4; -- 4 stratagem limit at level 65. Recharges every 60 seconds
+UPDATE `abilities_charges` SET `level` = 75 WHERE `recastId` = 231 AND `maxCharges` = 5; -- 5 stratagem limit at level 75. Recharges every 48 seconds
+
+-- RUN
+UPDATE `abilities` SET `level` = 71, `content_tag`= NULL WHERE `name` = 'liement';
+UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'one_for_all';
+
+-- GEO
+UPDATE `abilities` SET `level` = 65, `content_tag`= NULL WHERE `name` = 'concentric_pulse';
+UPDATE `abilities` SET `level` = 71, `content_tag`= NULL WHERE `name` = 'theurgic_focus'; -- TODO: Not yet implemented. TODO: Potency changed from +50MAB to +25MAB
+
+------------------------
+-- Disabled Abilities --
+------------------------
+DELETE FROM `abilities` WHERE `name`='majesty'; -- Majesty disabled
```

Comment:
```suggestion
UPDATE `abilities`
SET `level` = 76
WHERE `name` IN (
'majesty'
)
```

I believe weird issues can occur from deleting abilities. Most mobs will use mob abilities, but just to be safe.